### ISSUE TITLE
in POST /v2/entities using "multipolygon" geospatial type returns a error (http code: 500)

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2431]
in POST /v2/entities using "multipolygon" geospatial type returns a error (http code: 500)

#### Request
```
url: POST http://localhost:1026/v2/entities
headers:
    Content-Type: application/json
    Fiware-Service: test_geo_json
    Fiware-ServicePath: /test
payload: {"type": "house", "id": "room1",  "location":{"type": "geo:json", "metadata": {"very_hot": {"value": "hot", "type": "alarm"}}, "value": {"type": "MultiPolygon","coordinates": [[[[-75.69030761718749,35.74205383068037],[-75.5914306640625,35.74205383068037],[-75.69030761718749,35.285984736065735],[-75.69030761718749,35.74205383068037]],[[-109.0283203125, 36.98500309285596],[-109.0283203125,40.97989806962013],[-102.06298828125,37.00255267215955],[-109.0283203125,36.98500309285596]]]]}}}
```

#### Response
```
http code: 500 - Internal Server Error
headers:
   date: Wed, 03 Aug 2016 13:10:23 GMT
   fiware-correlator: a3197c82-597b-11e6-9af4-005056a20feb
   connection: Keep-Alive
   content-type: application/json
   content-length: 2574
payload: {"error":"InternalError","description":"Database Error (collection: orion-test_geo_json.entities - insert(): { _id: { id: "room1", type: "house", servicePath: "/test" }, attrNames: [ "location" ], attrs: { location: { type: "geo:json", creDate: 1470229823, modDate: 1470229823, value: { type: "MultiPolygon", coordinates: [ [ [ [ -75.69030761718749, 35.74205383068037 ], [ -75.5914306640625, 35.74205383068037 ], [ -75.69030761718749, 35.28598473606574 ], [ -75.69030761718749, 35.74205383068037 ] ], [ [ -109.0283203125, 36.98500309285596 ], [ -109.0283203125, 40.97989806962013 ], [ -102.06298828125, 37.00255267215955 ], [ -109.0283203125, 36.98500309285596 ] ] ] ] }, md: { very_hot: { type: "alarm", value: "hot" } }, mdNames: [ "very_hot" ] } }, creDate: 1470229823, modDate: 1470229823, location: { attrName: "location", coords: { type: "MultiPolygon", coordinates: [ [ [ [ -75.69030761718749, 35.74205383068037 ], [ -75.5914306640625, 35.74205383068037 ], [ -75.69030761718749, 35.28598473606574 ], [ -75.69030761718749, 35.74205383068037 ] ], [ [ -109.0283203125, 36.98500309285596 ], [ -109.0283203125, 40.97989806962013 ], [ -102.06298828125, 37.00255267215955 ], [ -109.0283203125, 36.98500309285596 ] ] ] ] } } } - exception: OperationException: { index: 0, code: 16755, errmsg: "Can't extract geo keys: { _id: { id: "room1", type: "house", servicePath: "/test" }, attrNames: [ "location" ], attrs: { location: { type: "geo:json",...", op: { _id: { id: "room1", type: "house", servicePath: "/test" }, attrNames: [ "location" ], attrs: { location: { type: "geo:json", creDate: 1470229823, modDate: 1470229823, value: { type: "MultiPolygon", coordinates: [ [ [ [ -75.69030761718749, 35.74205383068037 ], [ -75.5914306640625, 35.74205383068037 ], [ -75.69030761718749, 35.28598473606574 ], [ -75.69030761718749, 35.74205383068037 ] ], [ [ -109.0283203125, 36.98500309285596 ], [ -109.0283203125, 40.97989806962013 ], [ -102.06298828125, 37.00255267215955 ], [ -109.0283203125, 36.98500309285596 ] ] ] ] }, md: { very_hot: { type: "alarm", value: "hot" } }, mdNames: [ "very_hot" ] } }, creDate: 1470229823, modDate: 1470229823, location: { attrName: "location", coords: { type: "MultiPolygon", coordinates: [ [ [ [ -75.69030761718749, 35.74205383068037 ], [ -75.5914306640625, 35.74205383068037 ], [ -75.69030761718749, 35.28598473606574 ], [ -75.69030761718749, 35.74205383068037 ] ], [ [ -109.0283203125, 36.98500309285596 ], [ -109.0283203125, 40.97989806962013 ], [ -102.06298828125, 37.00255267215955 ], [ -109.0283203125, 36.98500309285596 ] ] ] ] } } } })"}
```

#### Expected
```
http code: 201 - Created
```

**Comment by iariasleon** [comment]
This issue is OK, the problem was in the defintion of the list of pairs of geographical coordinates. The following example specifies a GeoJSON MultiPolygon:
```
`{
  "type": "MultiPolygon",
  "coordinates": [
     [ [ [ -73.958, 40.8003 ], [ -73.9498, 40.7968 ], [ -73.9737, 40.7648 ], [ -73.9814, 40.7681 ], [ -73.958, 40.8003 ] ] ],
     [ [ [ -73.958, 40.8003 ], [ -73.9498, 40.7968 ], [ -73.9737, 40.7648 ], [ -73.958, 40.8003 ] ] ]
  ]
}
```

**Comment by fgalan** [comment]
(I'm clearing milestone, as this was not solved as part of 1.3.0)
